test/functional/os-install/install-download.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+
+# Author: John Akre
+# Email: john.w.akre@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment -e "$TEST_NAME" 10
+	create_bundle -n os-core -f /core "$TEST_NAME"
+
+}
+
+@test "INS015: Download os-install content without installing" {
+
+	run sudo sh -c "$SWUPD os-install $SWUPD_OPTS_NO_PATH --download $TARGETDIR" 
+
+	assert_status_is 0
+	expected_output=$(cat <<-EOM
+		Installing OS version 10 (latest)
+		Downloading packs for:
+		 - os-core
+		Finishing packs extraction...
+		Checking for corrupt files
+		No extra files need to be downloaded
+		Installation successful
```

Comment:
Please change this message so it shows something like "Installation files downloaded successfully" or something like that when `--download` is used